tests/test_jx/test_set_ops.py

```diff
@@ -46,7 +46,7 @@ def test_simplest(self):
                 "select": "a"
             },
             "expecting_list": {
-                "meta": {"format": "list"}, "data": ["b"]},
+                "meta": {"format": "list"}, "data": [{"a":"b"}]},
```

Comment:
this, and all the other in this file, are wrong.  If the select clause is an object (or string) then the query is requesting values.  If the select clause is an array, then the query is requesting objects with properties.